app/src/main/res/layout/widget_button.xml

```diff
@@ -10,44 +10,51 @@
     android:background="@drawable/widget_button_background"
     android:padding="4dp">
 
-    <LinearLayout
+    <RelativeLayout
         android:id="@+id/widgetImageButtonLayout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:paddingVertical="4dp"
+        android:layout_height="wrap_content"
+        android:paddingVertical="8dp"
+        android:paddingHorizontal="4dp"
         android:orientation="vertical">
 
         <ImageView
+            android:id="@+id/widgetImageButtonBackground"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:translationZ="-2dp"
```

Comment:
Why use `translationZ` which is relative, instead of `elevation`?

Reply:
> I believe they're equivalent? I'm not sure if there's any specific reason to use either.

They both work, but for static values [`elevation` is suggested while `translation` is suggested for animation](https://developer.android.com/develop/ui/views/theming/shadows-clipping#Elevation). And here, it actually seems to be useless as it is already the lowest item.

```suggestion
```